specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ServerConfigurationOptions.json

```diff
@@ -0,0 +1,264 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-08-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```

Comment:
>paths [](http://example.com/codeflow?start=3&length=5)

Please check the errors being flagged here and address
[https://github.com/Azure/azure-rest-api-specs/pull/21415/checks?check_run_id=9275154894](https://github.com/Azure/azure-rest-api-specs/pull/21415/checks?check_run_id=9275154894)

Reply:
Checked and had a discussion with @ericshape already. He said that these failures can be ignored as those rules that fail were recently added, and we only have failures in staging pipeline

Reply:
@rkmanda https://github.com/Azure/azure-openapi-validator/blob/develop/docs/resource-name-restriction.md is the new requirement that was added last month. All the swagger files from SQL RP are auto-generated. We are planning efforts to add the new swagger rules to the auto-generation process. But it needs time. can we ask for an exception for 2022-08-01-preview? thanks!